docs/guidance/identifiers.md

```diff
@@ -0,0 +1,38 @@
+# Project identifiers
+
+A project identifier is a globally unique identifier for an infrastructure project. Every project in OC4IDS has a project identifier in the `id` field.
+
+Project identifiers can be used to join up information published at different times or from different systems, for example, including a project identifier in contracting data enables information on the design, construction and supervision contracts for a single infrastructure project to be joined up.
```

Comment:
Use semi-colon instead of comma before "for example".

Reply:
> including a project identifier in contracting data makes it possible to join up data on the design, construction and supervision contracts within a single infrastructure project.

Move verb before object.

Reply:
Use 'data' not 'information' in first sentence.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/97/commits/6dbddcc20150fb09e92a8ab533ffde539d1f8ae7